labs/lab-2-javascript-game/script.js

```diff
@@ -0,0 +1,50 @@
+function startGame() {
+  const play = window.prompt("Do you want to play?");
+  if (play === "yes") {
+    var name = window.prompt("Enter your name:")
+    startCombat(name);
+  }
+}
+
+function getDamage() {
+  const min = 1;
+  const max = 5;
+  return (Math.floor(Math.random() * (max - min + 1) + min));
+}
+
+function startCombat(name) {
```

Comment:
You remember how I said functions should take one action, or answer one question? `startCombat` is a BIG function, and it does a lot of things; it sets up the game, it runs every attack round AND every battle round AND reports the final outcome. Can you break it into smaller, bite-sized pieces?